CHARTER.md

```diff
@@ -0,0 +1,104 @@
+
+*Third-party Organization:* A company or organization that uses MapLibre software for commercial or non-commercial purposes.
+
+## MapLibre Organization Goals
+The MapLibre Organization develops free open-source mapping libraries and tools for others to build applications and services.
+
+The primary goal of the MapLibre Organization is to build a friendly and innovative open-source community around the unaffiliated MapLibre Projects.
+
+The MapLibre Organization aims to encourage code development. We prefer to empower our users to build end-user applications, design maps, run map hosting services or distribute data rather than making it part of the MapLibre Projects.
+
+## Governance Bodies
+The MapLibre Organization has three Governance Bodies which support and represent the broader MapLibre Community:
+
+ * Technical Steering Committee
+ * Governing Board
+ * Voting Members
+
+### Technical Steering Committee
+The Technical Steering Committee discusses technical topics that affect multiple MapLibre projects, coordinates ongoing work, and decides if proposed changes with a major impact should be accepted or not.
+
+The committee meets on a monthly basis on a video call. 
+
+The Technical Steering Committee is open to everyone and there are no special requirements if someone wants to join it.
+
+Some projects may have their own steering committees to discuss issues relevant to just that project (e.g., native, gl-js, …).
+
+
+### Governing Board
+The Governing Board helps the Community to define our scope and direction, and is responsible for the organization finances.
+
+The Board is a group of 5 people who represent the MapLibre Organization.
+
+While the Governing Board’s purpose is not to handle technical topics in general, the Board retains the authority over all decisions the MapLibre Projects make, and may overrule the Technical Steering Committee in case of a dispute.
+
+No more than one person on the Governing Board shall be employed or contracted by a single Third-party organization.
+
+The Board shall have at least the following roles:
+
+ * Treasurer: Takes care of financial transactions and accounting.
+ * Secretary: Writes and publishes meeting notes, manages Voting membership.
+
+The Governing Board meets at least 4 times per year, publishes quarterly financial reports, and shares the content of their meetings with the MapLibre Community to the extent the Board deems appropriate.
+
+### Voting Members
+Voting Members are people who elect the Governing Board and who vote on changes to the MapLibre Charter.
+
+A precondition to becoming a Voting Member is that the person contributed in a non-trivial way or that they donated funds to the MapLibre Organization.
+
+If Voting Members are paid by a Third-party organization for their work on the MapLibre Projects, they shall declare publicly who their employer is. At most 10 percent of the Voting Members shall be employed by a single Third-party organization.
```

Comment:
Thinking a bit about how to manage this when Voting members join a 3P organization... The default (and probably right) situation is for their Voting seat to be put on hold for the duration of their employment. To remain a Voting member in good standing, they'd coordinate with existing Voting members affiliated with that 3P offline (not a public consideration).

Did the board discuss the distinction between personal and professional involvement of 3P employees and come to any conclusions? [(Context](https://github.com/maplibre/maplibre/discussions/32#discussioncomment-3149796)]

Reply:
> Did the board discuss the distinction between personal and professional involvement of 3P employees and come to any conclusions? https://github.com/maplibre/maplibre/discussions/32#discussioncomment-3149796]

Yes, this was discussed, but I think there's still some ambiguity in the charter (obviously) and differing opinions on this.

I _think_ the best option so far is:

1) Apply the absolute rule to the governing board (no more than one member can be from one organization, regardless of professional or private contributions).
2) Apply a flexible rule to the voting membership: only professional members are counted against the 10%.

Reply:
I asked all the Voting Members in private the following question:

> Would you like to declare that you are paid to work on MapLibre by a company?

And I collected the answers an will publish the next days. Some people clearly feel that they are paid to work on MapLibre related things. Ohers might work in a maps organization but their day job has nothing to do with any of the MapLibre Projects.

Reply:
In the end it is not so difficult to find out the employer of a person based on their GitHub handle. And we are still a small community.

One way we could go is that we write in the Charter that Voting Members have the right to declare a company that pays them to work on MapLibre related stuff. This allows them to show transparently to the broader community where they come from and who's interests they might represent...

Reply:
But yeah, it is a mess with that threshold. What I just want to avoid is that we get into a situation where one large organization has enough Voting Members to effectively determine alone who is on the Governing Board. The cool thing about open-source however is that people are free to walk away and fork the code. We always wanted and still want to avoid community fragmentation because on fork is better than many forks...

Reply:
The declared company affiliations were published in #95

Reply:
Added a note that Voting Members shall inform the MapLibre Organization if their work situation changes...

Reply:
Closing this for now. If we need more detailed instructions how to handle the situation when too many existing Voting Members join one company, feel free to suggest a change.